layouts/partials/extend_footer.html

```diff
@@ -3,19 +3,32 @@
 </a>
 
 <div class="mailing-list-container">
+  <script src="https://f.convertkit.com/ckjs/ck.5.js"></script>
```

Comment:
Annoying to load an external script. :disappointed:
But good enough to get the initial integration going.

Reply:
When testing on DuckDuckGo mobile, subscription still works even though the script is blocked. However, the user is redirected to a generic ConvertKit page, which isn't a great experience. 

Should either: (1) self-host the script; (2) keep the script but have a better setup for redirection after posting; or (3) put together a minimal self-hosted version of the script.

In any case, it's good enough for now.